src/components/BookList.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import { Text, View, FlatList } from 'react-native';
+import { connect } from 'react-redux';
+import axios from 'axios';
+
+import * as actionCreators from '../actions';
+import * as statusCodes from '../reducers/constants';
+
+import styles from './styles/bookList';
+import BookItem from './BookItem';
+import { Spinner } from './common';
+
+class BookList extends Component {
+  componentDidMount() {
+    this.props.requestBooks();
+  }
+
+  keyExtractor = item => item.id.toString();
+  renderItem = ({ item }) => <BookItem book={item} />;
+
+  List = () => (
+    <View style={styles.view}>
+      <FlatList data={this.props.books} renderItem={this.renderItem} keyExtractor={this.keyExtractor} />
+    </View>
+  );
+
+  ErrorComponent = () => (
+    <View style={styles.centered}>
+      <Text>Connection Error </Text>
+    </View>
+  );
+
+  render() {
+    if (this.props.status === statusCodes.STATUS_WAIT) {
+      return <Spinner />;
+    }
+    if (this.props.status === statusCodes.STATUS_OK) {
+      return <this.List />;
+    }
+    return <this.ErrorComponent />;
+  }
+}
+
+const mapStateToProps = state => ({
+  books: state.booksStore.data.map(book => {
+    if (!book.thumbnail) {
+      return { ...book, thumbnail: require('./img/descarga.jpeg') };
```

Comment:
Try importing this with import at the top of your file:
import _imageName_ from './img/descarga.jpeg';